src/main/java/games/strategy/util/PropertyUtil.java

```diff
@@ -90,11 +90,10 @@ private static Field getPropertyField(final String propertyName, final Object su
     }
   }
 
-
-  private static String capitalizeFirstLetter(final String aString) {
-    char first = aString.charAt(0);
+  private static String capitalizeFirstLetter(final String str) {
```

Comment:
nit: prefer `str` to be spelled out, or renamed dto someting like `input`, but NBD